definitions/infra-azureappliactioninsight/golden_metrics.yml

```diff
@@ -0,0 +1,36 @@
+browserTimingsReceiveDuration:
```

Comment:
Minor typo on the folder name here: s/infra-azureappliactioninsight/infra-azureapplicationinsight/g